Unity-Technologies/ScriptableRenderPipeline
# removed test folder that was readded by bad merge.

### Purpose of this PR
 A bad merged readded files that were deleted to the repo. This PR removes them. 

---
### Release Notes
 Not needed.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
- [x ] Opened test project + Run graphic tests locally
- [x ] Checked new UI names with UX convention
- [x ] Tested UI multi-edition + Undo/Redo
- [x ] C# and shader warnings (supress shader cache to see them)
- Other: 

**Automated Tests**: Run tests locally.

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None

**Halo Effect**: None

---
### Comments to reviewers
